### ISSUE TITLE
support for `pull` on disk.frame?

### ISSUE THREAD

**Issue by cboettig** [xiaodaigh/disk.frame#278]
In `dplyr`-land with remote sources, `pull()` implies `compute()`. `pull()` doesn't seem to be implemented for a `disk.frame` though.   i.e. it would be nice to do:

```r

library(disk.frame)
library(nycflights13)
library(dplyr)

df <- as.disk.frame(flights)
df %>% pull(carrier)

```

Currently this only works if we `collect()` the whole `df` first.  Ideally this would work without having to call `collect()`, and would not read in the whole `df` but only the individual column requested (I believe `fst` supports parsing of a single column).  

Thanks!

**Comment by xiaodaigh** [comment]
Hmm, I missed this. Thanks for the report! Much appreciated.

I have now implemented it in #279. On CRAN soon. All of these should work

```r
library(disk.frame)
setup_disk.frame()

library(nycflights13)
# library(dplyr) # not needed

diskf <- as.disk.frame(flights)
diskf %>%
   srckeep("carrier") %>%
   pull(carrier)

diskf %>%
   srckeep("carrier") %>%
   pull(2)

diskf %>%
   srckeep("carrier") %>%
   pull(-1)
````

> would not read in the whole `df`

Until I revamp the backend for disk.frame( which will only come after I fix the NSE bugs and increase test coverage to close to 100%) the only way to load only the columns you need is via `srckeep` (for source keep)

```r
library(disk.frame)
setup_disk.frame()

library(nycflights13)
library(dplyr)
df <- as.disk.frame(flights)
df %>%
   srckeep("carrier") %>%
   pull(carrier)
````

**Comment by xiaodaigh** [comment]
disk.frame 0.3.5 is CRAN and it has the `pull` verb implemented
